src/assets/scripts/client/constants/gameOptionConstants.js

```diff
@@ -34,6 +35,16 @@ export const GAME_OPTION_VALUES = [
         ]
     },
     {
+        name: GAME_OPTION_NAMES.PTL_LENGTH,
+        defaultValue: '1',
+        description: 'Projected Track Line (PTL) Length',
+        type: 'select',
+        data: [
+            ['1', '1'],
+            ['0.5', '0.5']
```

Comment:
on `ksfo`, a length of `2` will cause the PTL from an arrival in the southeast to extend clear across _the entire_ airspace.